packages/dd-trace/src/config.js

```diff
@@ -901,6 +914,7 @@ class Config {
       calc.isIntelligentTestRunnerEnabled && !isFalse(this._isCiVisibilityGitUploadEnabled()))
     this._setBoolean(calc, 'spanComputePeerService', this._getSpanComputePeerService())
     this._setBoolean(calc, 'stats.enabled', this._isTraceStatsComputationEnabled())
+    this._setBoolean(calc, 'apmTracingEnabled', !this._isAppsecStandaloneEnabled())
```

Comment:
at the moment the only product that wants apm tracing disabled is appsec standalone but there could be more in the future.